spec/spec_tests/seed_list_discovery_spec.rb

```diff
@@ -78,6 +82,8 @@
           end
 
           if test.expected_options
+            # In Evergreen sometimes this test fails intermittently.
+            retry_test
```

Comment:
Can I remove this one since it's on the context on line 57?

Reply:
Yes I think so, I've done this.